packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -379,23 +380,28 @@ export async function scanParentDirs(
       'pnpm-lock.yaml',
       'bun.lock',
       'bun.lockb',
+      'vlt-lock.json',
     ],
   });
   let lockfilePath: string | undefined;
   let lockfileVersion: number | undefined;
   let cliType: CliType;
 
   const bunLockPath = bunLockTextPath ?? bunLockBinPath;
-  const [packageLockJson, pnpmLockYaml, bunLock, yarnLock] = await Promise.all([
-    npmLockPath
-      ? readConfigFile<{ lockfileVersion: number }>(npmLockPath)
-      : null,
-    pnpmLockPath
-      ? readConfigFile<{ lockfileVersion: number }>(pnpmLockPath)
-      : null,
-    bunLockPath ? fs.readFile(bunLockPath) : null,
-    yarnLockPath ? fs.readFile(yarnLockPath, 'utf8') : null,
-  ]);
+  const [packageLockJson, pnpmLockYaml, bunLock, yarnLock, vltLock] =
+    await Promise.all([
+      npmLockPath
+        ? readConfigFile<{ lockfileVersion: number }>(npmLockPath)
+        : null,
+      pnpmLockPath
+        ? readConfigFile<{ lockfileVersion: number }>(pnpmLockPath)
+        : null,
+      bunLockPath ? fs.readFile(bunLockPath) : null,
+      yarnLockPath ? fs.readFile(yarnLockPath, 'utf8') : null,
+      vltLockPath
+        ? readConfigFile<{ lockfileVersion: number }>(vltLockPath)
```

Comment:
I don't think the vlt lockfile has a `lockfileVersion` field

Reply:
Good shout I've removed.